public_html/composer.json

```diff
@@ -0,0 +1,129 @@
+					"version": "1.0.0",
+					"source": {
+						"type": "svn",
+						"url": "https://plugins.svn.wordpress.org/tbz-camptix-paystack/",
+						"reference": "tags/1.0.0/"
+					}
+				},
+				{
+					"name": "wordpress-plugin/camptix-paynow",
+					"type": "wordpress-plugin",
+					"version": "1.0.4",
+					"source": {
+						"type": "svn",
+						"url": "https://plugins.svn.wordpress.org/gateway-camptix-paynow-payment/",
+						"reference": "tags/1.0.4/"
+					}
+				},
+				{
+					"name": "wordpress-plugin/json-rest-api",
+					"type": "wordpress-plugin",
+					"version": "1.2.5",
+					"source": {
+						"type": "svn",
+						"url": "https://plugins.svn.wordpress.org/json-rest-api/",
+						"reference": "tags/1.2.5/"
+					}
+				},
+				{
+					"name": "wordpress-meta/wporg-profiles-wp-activity-notifier",
+					"type": "wordpress-plugin",
+					"version": "1.1",
+					"source": {
+						"type": "svn",
+						"url": "https://meta.svn.wordpress.org/sites/trunk/wordpress.org/public_html/wp-content/plugins/",
+						"reference": "wporg-profiles-wp-activity-notifier/"
+					}
+				}
+			]
+		}
+	],
+	"require": {
+		"league/oauth2-client": "2.4.1"
+	},
+	"require-dev": {
+		"wpackagist-plugin/akismet": "4.1.2",
+		"wordpress-plugin/bbpress": "2.6.*",
+		"wpackagist-plugin/camptix": "dev-trunk",
+		"wpackagist-plugin/campt-indian-payment-gateway": "1.8",
+		"wordpress-plugin/camptix-bd-payments": "1.2",
+		"wpackagist-plugin/camptix-mercadopago": "1.0.6",
+		"wpackagist-plugin/camptix-pagseguro": "1.5.5",
+		"wpackagist-plugin/camptix-payfast-gateway": "1.0.0",
+		"wpackagist-plugin/camptix-trustcard": "1.0.4",
+		"wpackagist-plugin/camptix-trustpay": "1.0.5",
+		"wordpress-plugin/camptix-paystack": "1.0.0",
+		"wordpress-plugin/camptix-paynow": "1.0.4",
+		"wpackagist-plugin/classic-editor": "dev-trunk",
+		"wpackagist-plugin/custom-content-width": "1.0.1",
+		"wpackagist-plugin/edit-flow": "0.9",
+		"wpackagist-plugin/email-post-changes": "dev-trunk",
+		"wpackagist-plugin/gutenberg": "6.0.0",
+		"wpackagist-plugin/hyperdb": "dev-trunk",
+		"wpackagist-plugin/jetpack": "7.5.1",
+		"wordpress-plugin/json-rest-api": "1.2.5",
+		"wpackagist-plugin/liveblog": "1.9.4",
+		"wpackagist-plugin/public-post-preview": "2.8.0",
+		"wpackagist-plugin/pwa": "0.2.0",
+		"wpackagist-plugin/supportflow": "dev-trunk",
+		"wpackagist-plugin/tagregator": "dev-trunk",
+		"wpackagist-plugin/wordpress-importer": "0.6.4",
+		"wpackagist-plugin/wp-super-cache": "1.6.8",
+		"wordpress-meta/wporg-profiles-wp-activity-notifier": "1.1",
+		"wpackagist-theme/p2": "1.5.8",
+		"wpackagist-theme/twentyten": "2.9",
+		"wpackagist-theme/twentyeleven": "3.3",
+		"wpackagist-theme/twentytwelve": "3.0",
+		"wpackagist-theme/twentythirteen": "2.9",
+		"wpackagist-theme/twentyfourteen": "2.7",
+		"wpackagist-theme/twentyfifteen": "2.5",
+		"wpackagist-theme/twentysixteen": "2.0",
+		"wpackagist-theme/twentyseventeen": "2.2",
+		"wpackagist-theme/twentynineteen": "1.4"
+	}
```

Comment:
`MIGHT` - Core has a `scripts` section too - https://github.com/WordPress/wordpress-develop/blob/e6b7205/composer.json

Maybe we should add phpunit/phpcs/phpmd here?

Reply:
Definitely worth exploring for an iteration. I don't think we should include it in this PR, but I'll open a separate issue.